### ISSUE TITLE
Insufficient color contrast accessibility errors inside the navigation bar

### ISSUE THREAD

**Issue by DemeSzabolcs** [StartBootstrap/startbootstrap-bare#58]
The theme has color contrast accessibility errors.
This is just one of them, but there should be more inside the navigation bar: 

```
1: Elements must have sufficient color contrast
Description: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds
Help: Elements must have sufficient color contrast
Help URL: https://dequeuniversity.com/rules/axe/4.2/color-contrast?application=axeAPI
Impact: serious
Tags: cat.color, wcag2aa, wcag143
Element(s):
Html:
<a class="nav-link dropdown-toggle" id="navbarDropdown" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">Dropdown</a>

Selector(s):
.nav-link

To solve:
Fix at least one of the following issues:

SERIOUS: Element has insufficient color contrast of 1.72 (foreground color: #2424db, background color: #212529, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
```
